network/src/protocols/wire/messaging/v1/mod.rs

```diff
@@ -46,10 +58,10 @@ pub type Priority = u8;
 
 #[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
 pub struct RpcRequest {
-    /// RequestId for the RPC Request.
-    pub request_id: RequestId,
     /// `protocol_id` is a variant of the ProtocolId enum.
     pub protocol_id: ProtocolId,
+    /// RequestId for the RPC Request.
+    pub request_id: RequestId,
```

Comment:
want to piggy back this breaking change too? lol

Reply:
woops, meant to call this out :D

Reply:
Is the reason for this that you can send out the ProtocolId related from reading the first bytes off the wire as we expect them to be more common than request_id related errors?

Reply:
You answered this proactively in the updated summary.  Thanks!